.github/workflows/largemodel_unit_test_CI.yml

```diff
@@ -62,6 +62,12 @@ jobs:
           repository: marqo-ai/marqo-base
           path: marqo-base
 
+      - name: Install FFmpeg
```

Comment:
What alternatives did we consider for this package? Why did we pick this?

Reply:
Action item for later: find a way to run these tests inside the marqo-base image so that we don't need to install these deps in each build